commons/passive/audio/phone_audio_input.avsc

```diff
@@ -0,0 +1,24 @@
+{
+    "namespace": "org.radarcns.passive.audio",
+    "type": "record",
+    "name": "PhoneAudioInput",
+    "doc": "Uncompressed high-quality audio data collected by the PhoneAudioInput plugin, making use of low-level classes that interact directly with hardware.",
+    "fields": [
+        { "name": "time", "type": "double", "doc": "Device timestamp in UTC (s)." },
+        { "name": "timeReceived", "type": "double", "doc": "Device receiver timestamp in UTC (s)." },
+        { "name": "fileName", "type": "string", "doc": "Name of the audio file after it is saved to S3." },
+        { "name": "filePath", "type": "string", "doc": "Path of the audio file retrieved after uploading to S3 storage." },
+        { "name": "deviceName", "type": "string", "doc": "Name of the input audio device used for routing during this recording." },
+        { "name": "deviceId", "type": "string", "doc": "Identifier associated with the input device used for audio recording." },
+        { "name": "deviceSampleRates", "type": "string", "doc": "Supported sample rates of the input audio device." },
```

Comment:
good point, @this-Aditya is this supposed to be an array or will it be like a comma-separated string?

Reply:
Initially, I planned to use an array for this, but I remember [joris's comment](https://github.com/RADAR-base/RADAR-Schemas/pull/352#discussion_r1299812336) to avoid using an array here.